src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -365,15 +365,16 @@ LedgerMaster::getFullValidatedRange (std::uint32_t& minVal, std::uint32_t& maxVa
     if (!maxVal)
         return false;
 
+    boost::optional<std::uint32_t> maybeMin;
```

Comment:
Hm. That's a good idea. There's no reason the parameters need to be passed in to start with. Same could be said for `getValidatedRange`.